source/agora/network/NetworkClient.d

```diff
@@ -201,15 +201,12 @@ class NetworkClient
         import agora.common.Hash;
         const tx_hash = tx.hashFull();
 
-        this.taskman.runTask(
-        {
-            // if the node already has this tx, don't send it
-            if (this.attemptRequest!(API.hasTransactionHash, Throw.No,
-                LogLevel.Trace)(this.api, tx_hash))
-                return;
+        // if the node already has this tx, don't send it
```

Comment:
Should consider squashing this with the code that introduces the RequestQueue, otherwise this commit might break some assumptions.